internal/env/load.go

```diff
@@ -0,0 +1,13 @@
+package env
+
+import (
+	"github.com/joho/godotenv"
+	"github.com/rs/zerolog/log"
```

Comment:
not the logger from the previous stack?

Reply:
cannot be sure that env vars exist here if we're using dotenv